releasenotes/notes/fix-716-mismatch-dimension-pytorch-ba01bea90eba1435.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixes the dimension mismatch error in the `torch_connector` raised when using other-than 3D datasets.
+    The updated implementation defines the Einstein summation signature dynamically based on the number of
+    dimensions `ndim` of the input data (up to 26 dimensions).
```

Comment:
Should we also note this in the TorchConnector somewhere. Maybe in its main docstring that datasets are limit to 26 dimensions and any larger with raise an error.

Reply:
I agree it would be useful to mention it in the main docstring. Now added.